packages/plugin-patch/sources/PatchResolver.ts

```diff
@@ -6,7 +6,7 @@ import * as patchUtils
 
 // We use this to for the patches to be regenerated without bumping the whole
 // cache, like when the libzip had incorrect mtime in some cases
-const CACHE_VERSION = 2;
+const CACHE_VERSION = 3;
```

Comment:
Rather than this cache version, we should instead bump the global one in `Cache.ts` (because the cache generation changes also affect archives generated through git dependencies, etc).

Reply:
Aah, right. Updated.